File: js/bot.js
function Bot() {
  Paddle.call(this);

  this.x = game.width - this.width - 20;
}

Bot.prototype = Object.create(Paddle.prototype);
Bot.prototype.constructor = Bot;

Bot.prototype.update = function() {
  var speed = 5;

  if (this.y < game.ball.y - this.height) {
    this.yVelocity = speed;
  } else if (this.y > game.ball.y) {
    this.yVelocity = -speed;
  }

  Paddle.prototype.update.apply(this, arguments);
};

File: app/models/bot.rb
class Bot < Paddle

  def initialize
    super
    @x = @game.width - @width - 20
  end

  def update
    speed = 5;

    if @y < @game.ball.y - @height
      @y_velocity = speed
    elsif @y > @game.ball.y
      @y_velocity = -speed
    end

    super
  end

end
